demos/python_demos/bert_question_answering_embedding_demo/models.lst

```diff
@@ -0,0 +1,5 @@
+# This file can be used with the --list option of the model downloader.
+# For the conventional (optional) BERT model, that is used to calculate the final answer from the the contexts returned from the embedding-only model (below)
+#     the demo should support any BERT calibrated/fine-tuned/distilled/etc on the SQuAD dataset, e.g. used in the basic BERT demo (../bert_question_answering_demo)
```

Comment:
You should add the other BERT models to this file, as well.

Reply:
somebody already done it

Reply:
What do you mean? I only see the embedding model.

Reply:
Sorry. I've misunderstood you comment